app/renderer/components/Inspector/Screenshot.js

```diff
@@ -34,12 +34,87 @@ export default class Screenshot extends Component {
     });
   }
 
-  async handleScreenshotClick () {
-    const {screenshotInteractionMode, applyClientMethod,
-           swipeStart, swipeEnd, setSwipeStart, setSwipeEnd} = this.props;
-    const {x, y} = this.state;
 
-    if (screenshotInteractionMode === 'tap') {
+  clickScreenshotElement (source, selectedElement, scaleRatio, x, y, highlighterXOffset, highlighterYOffset) {
```

Comment:
A JSDoc would be helpful here explaining what the function is doing and what the parameters are.